### DISCUSSION TITLE
OpenAI compatible server

### DISCUSSION THREAD

**Discussion by nacho00112** [microsoft/onnxruntime-genai#419] [New features / APIs]
I need one

**Comment by yufenglee** [comment]
It is underway. Will update this thread once it is ready.

**Comment by natke** [comment]
Moving this to a discussion of new features

**Reply by bpietroiu** [reply]
See here for my take on this 
https://github.com/microsoft/onnxruntime-genai/discussions/995

**Comment by kunal-vaishnavi** [comment]
You can try out [Foundry Local](https://foundrylocal.ai/), which is powered by ONNX Runtime GenAI.
